modules/ROOT/partials/depl-examples/ubuntu-compose/shared-setup.adoc

```diff
@@ -439,4 +455,82 @@ If no password can be identified, you must reset the admin password via the comm
 
 To change the admin password from the command line, which you can do at any time, follow the guide described in xref:deployment/general/general-info.adoc#password-reset-for-the-admin-user[Password Reset for the Admin User].
 
+== Volume Migration
+
+This section gives some guidance if you want to migrate the Infinite Scale docker internal volumes to docker volumes using a local path. This can be required like to separate the container from its data or if a high data volume is expected. See additional documentation in the xref:deployment/tips/useful_mount_tip.adoc[Start a Service After a Resource is Mounted] if you want to use network mounts like NFS or iSCSI for the data directory.
```

Comment:
```suggestion
This section gives some guidance if you want to migrate the Infinite Scale docker internal volumes to docker volumes using a local path. For example, this can be required to separate the container from its data or if a high data volume is expected. See additional documentation in the xref:deployment/tips/useful_mount_tip.adoc[Start a Service After a Resource is Mounted] if you want to use network mounts like NFS or iSCSI for the data directory.
```